Activities/Prac 5/emails.py

```diff
@@ -0,0 +1,33 @@
+"""
+CP1404 Practical 5
+Email list
+"""
+
+
+def main():
+    emails_and_names = {}
+    user_email = input("Email: ").lower()
+    while user_email != "":
+        if user_email not in emails_and_names:
+            user_name = get_name(user_email)
+            emails_and_names = {user_email: user_name}
```

Comment:
Consider changing:
emails_and_names = {user_email: user_name} is overwriting the dictionary not adding to it.
try:
emails_and_names[user_email] = user_name

Reply:
thanks, changed it